plugins/signalmonitor/signalhistorymodel.cpp

```diff
@@ -156,14 +172,18 @@ QVariant SignalHistoryModel::data(const QModelIndex &index, int role) const
 
 QVariant SignalHistoryModel::headerData(int section, Qt::Orientation orientation, int role) const
 {
-    if (role == Qt::DisplayRole && orientation == Qt::Horizontal) {
-        switch (section) {
-        case ObjectColumn:
-            return tr("Object");
-        case TypeColumn:
-            return tr("Type");
-        case EventColumn:
-            return tr("Events");
+    if (orientation == Qt::Horizontal) {
```

Comment:
the two conditionals can be merged like before, resulting in a decreased indent-depth